<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>littleMermaidGame</class>
 <widget class="QMainWindow" name="littleMermaidGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="pagePlay">
     <widget class="QPushButton" name="pushButtonPlay">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>400</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLogIn">
     <widget class="QLabel" name="labelHeading1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Don't lose your progress!</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Save your game and continue playing right </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>where you left off.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreateAccount">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>CREATE ACCOUNT</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Already have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogIn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>LOG IN</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageCreateAccount">
     <widget class="QLabel" name="labelHeading5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Create your account!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreateMyAccount">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>CREATE MY ACCOUNT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackCreate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>189</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelUsername">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassword">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>241</y>
        <width>67</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>268</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelNickname">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>321</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nickname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNickname">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>349</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLogInScreen">
     <widget class="QPushButton" name="pushButtonLogIntoAccount">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>LOG IN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonForgotYourPassword">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Forgot your password?</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Log in to your Account!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackToLogin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsernameEnter">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelUsernameEnter">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPasswordEnter">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>67</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPasswordEnter">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageForgotPassword">
     <widget class="QPushButton" name="pushButtonResetPassword">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>RESET PASSWORD</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Forgot your password?</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Don't worry. Enter your username and password</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeading9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>in order to reset your password.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackForgot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPasswordForgot">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>350</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPasswordForgot">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>98</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>NEW Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsernameForgot">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelUsernameForgot">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>67</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLevels">
     <widget class="QPushButton" name="pushButtonLevel1">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>510</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
font: 14pt &quot;MS Shell Dlg 2&quot; rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLevel2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLevel3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonUsersname">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLogOut">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Log Out</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageUserScoreBoard">
     <widget class="QPushButton" name="pushButtonBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewMyTopScores">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>341</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLevel1">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
